schemas/examples.ipldsch.json

```diff
@@ -1,67 +1,3 @@
 {
-	"schema": {
-		"ExampleWithNullable": {
-			"kind": "map",
-			"keyType": "String",
-			"valueType": {
-				"kind": "link"
-			},
-			"valueNullable": true
-		},
-		"ExampleWithAnonDefns": {
-			"kind": "struct",
-			"fields": {
-				"fooField": {
-					"type": {
-						"kind": "map",
-						"keyType": "String",
-						"valueType": "String"
-					},
-					"optional": true
-				},
-				"barField": {
-					"type": {
-						"kind": "map",
-						"keyType": "String",
-						"valueType": "String"
-					},
-					"nullable": true
-				},
-				"bazField": {
-					"type": {
-						"kind": "map",
-						"keyType": "String",
-						"valueType": "String",
-						"valueNullable": true
-					}
-				},
-				"wozField": {
-					"type": {
-						"kind": "map",
-						"keyType": "String",
-						"valueType": {
-							"kind": "list",
-							"valueType": "String",
-							"valueNullable": true
-						}
-					}
-				},
-				"boomField": {
-					"type": {
-						"kind": "link",
-						"expectedType": "ExampleWithNullable"
-					}
-				}
-			},
-			"representation": {
-				"map": {
-					"fields": {
-						"fooField": {
-							"rename": "foo_field"
-						}
-					}
-				}
-			}
-		}
-	}
+  "error": "This document has moved: https://ipld.io/docs/schemas/"
```

Comment:
The link here does not take you to an actual example of a json representation of a schema. This is strictly less useful.